src/main/java/org/ohdsi/circe/cohortdefinition/CohortExpressionQueryBuilder.java

```diff
@@ -435,6 +438,8 @@ public String buildExpressionQuery(CohortExpression expression, BuildExpressionQ
         resultSql = StringUtils.replace(resultSql, "@target_database_schema.@target_cohort_table", options.targetTable);
       if (options.resultSchema != null)
         resultSql = StringUtils.replace(resultSql, "@results_database_schema", options.resultSchema);
+      if (options.vocabularySchema != null)
```

Comment:
This should be changed so that the vocabulary schema is optional.  If you have a CDM schema and the vocabulary shema is null, then replace vocabulary schema with the cdmSchema value.